pkg/condition/condition.go

```diff
@@ -0,0 +1,149 @@
+package condition
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+
+	operators "github.com/operator-framework/api/pkg/operators/v2"
+	"github.com/pkg/errors"
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+	kubeTypes "k8s.io/apimachinery/pkg/types"
+	kubeWait "k8s.io/apimachinery/pkg/util/wait"
```

Comment:
We only need to introduce prefixes if there is a collision. I suggest leaving `kube` out.